devel/mos/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+categories          devel
+license             Apache-2.0
+name                mos
+
+set python.versions \
+                    38
+
+depends_lib-append  port:libftdi1 \
+                    port:libftdi0 \
+                    port:libusb \
+                    port:libusb-compat \
```

Comment:
Does this really require both libusb and libusb-compat?

A libusb-devel port exists, so the libusb dependency should be written as `path:lib/libusb-1.0.dylib:libusb`